programs/local/chdb.h

```diff
@@ -18,6 +18,7 @@ struct CHDB_EXPORT local_result
     double elapsed;
     uint64_t rows_read;
     uint64_t bytes_read;
+    char * error_message;
```

Comment:
This will be a breaking change for all bindings.
Let make all function and struct a `v2` version.